108892_CEVersion_New_Create_Option_CECostSource

VarunAgrawalJKT opened the pull request:
Cost will be calculated as per the cost source.

File changed:
est\EstimateCalcProcs.p
sys\inc\create_nk1.p
ref\sys-ctrl.i

VarunAgrawalJKT committed:
108892_CEVersion_New_Create_Option_CECostSource

Cost will be calculated as per the cost source.

File changed:
est\EstimateCalcProcs.p
sys\inc\create_nk1.p
ref\sys-ctrl.i

brad-advantzware reviewed (changes requested):
Varun - VendorCostThen... Should still look at the vendor cost. table.  It is difficult to tell from this since there are a lot of indentation changes, but it looks like the "GetVendorCost" is only called if it is vendorCostOnly.  This is not correct.  We simply needed to remove the "RETURN".
If the cost is still 0 from the GetVendorCost process, *then* you see if there is a VendorCostThen... option to overwrite the 0 cost.

VarunAgrawalJKT committed:
108892_CEVersion_New_Create_Option_CECostSource

Code review points have been implemented.

File Changed:
est\EstimateCalcProcs.p

VarunAgrawalJKT committed:
108892_CEVersion_New_Create_Option_CECostSource

File Changed:
sys/inc/create_nk1.p

brad-advantzware committed:
Merge branch 'develop' into feature/108892_CEVersion_New_Create_Option_CECostSource

# Conflicts:
#	Source/est/EstimateCalcProcs.p
#	Source/sys/inc/create_nk1.p
